scripts/zones/Yuhtunga_Jungle/Zone.lua

```diff
@@ -17,8 +17,8 @@ zoneObject.onChocoboDig = function(player, precheck)
 end
 
 zoneObject.onInitialize = function(zone)
-    UpdateNMSpawnPoint(ID.mob.MEWW_THE_TURTLERIDER)
-    GetMobByID(ID.mob.MEWW_THE_TURTLERIDER):setRespawnTime(math.random(900, 10800))
+    -- NM Persistence
+    xi.mob.nmTODPersistCache(zone, ID.mob.MEWW_THE_TURTLERIDER)
```

Comment:
If you want to do the WOTG also you can do one for : https://ffxiclopedia.fandom.com/wiki/Bayawak
Same as KV the localvar should be changed to persist and then in the zone lua use that server var for the weatherchange function